kiwix-desktop.pro.user

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.pro.user file is local for every developer's machine, no need to commit it to repository